tests/manager/sqlalchemy/model/test_work.py

```diff
@@ -1411,6 +1407,171 @@ def test_to_search_doc_datetime_cases(self, db: DatabaseTransactionFixture):
+        )[0]
+        search_doc_work2 = list(
+            filter(lambda x: x["work_id"] == work2.id, search_docs)
+        )[0]
+
+        def compare(doc: dict[str, Any], work: Work) -> None:
+            assert doc["_id"] == work.id
+            assert doc["work_id"] == work.id
+            assert doc["title"] == work.title
+            assert doc["sort_title"] == work.sort_title
+            assert doc["subtitle"] == work.subtitle
+            assert doc["series"] == work.series
+            assert doc["series_position"] == work.series_position
+            assert doc["language"] == work.language
+            assert doc["author"] == work.author
+            assert doc["sort_author"] == work.sort_author
+            assert doc["medium"] == work.presentation_edition.medium
+            assert doc["publisher"] == work.publisher
+            assert doc["imprint"] == work.imprint
+            assert (
+                doc["permanent_work_id"] == work.presentation_edition.permanent_work_id
+            )
+            assert doc["presentation_ready"] == work.presentation_ready
+            assert doc["last_update_time"] == work.last_update_time
+            assert doc["fiction"] == "Fiction" if work.fiction else "Nonfiction"
+            assert doc["audience"] == work.audience
+            assert doc["summary"] == work.summary_text
+            assert doc["quality"] == work.quality
+            assert doc["rating"] == work.rating
+            assert doc["popularity"] == work.popularity
+
+            if work.suppressed_for:
+                assert doc["suppressed_for"] == [l.id for l in work.suppressed_for]
+            else:
+                assert doc["suppressed_for"] is None
+
+            if work.license_pools:
+                assert len(doc["licensepools"]) == len(work.license_pools)
+                for idx, pool in enumerate(work.license_pools):
+                    actual = doc["licensepools"][idx]
+                    assert actual["licensepool_id"] == pool.id
+                    assert actual["data_source_id"] == pool.data_source_id
+                    assert actual["collection_id"] == pool.collection_id
+                    assert actual["open_access"] == pool.open_access
+                    assert actual["suppressed"] == pool.suppressed
+            else:
+                assert doc["licensepools"] is None
+
+            if work.custom_list_entries:
+                assert len(doc["customlists"]) == len(work.custom_list_entries)
+                for idx, custom_list in enumerate(work.custom_list_entries):
+                    actual = doc["customlists"][idx]
+                    assert actual["list_id"] == custom_list.list_id
+                    assert actual["featured"] == custom_list.featured
+            else:
+                assert doc["customlists"] is None
+
+            if work.presentation_edition.contributions:
+                assert len(doc["contributors"]) is len(
```

Comment:
'==' instead `is` here?

Reply:
This whole test was just moved in from the coverage providers tests. Not sure why it was using `is`. I updated it.